src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/Tov.cpp

```diff
@@ -99,6 +103,14 @@ void lindblom_rhs(
       d_log_conformal_factor = -0.25 * d_mass_over_radius;
     }
   } else {
+    // This statement is triggered by one of the test examples,
+    // Test_Tov.cpp: void {anonymous}::test_rueter()
+    // Test_Tov.cpp: void {anonymous}::test_baumgarte_shapiro()
+    // not sure what to do about that
```

Comment:
@kidder any idea what's happening here?

Reply:
The naive answer seems to be that there is a place inside of the star where 2M/r > 1, this might be a distinction between isotropic radius and areal radius

Reply:
Ah yea, that's probably what it is. That the condition only applies in Schwarzschild-like coordinates

Reply:
Hm I changed the assert statement to only run when the coordinates as Schwarzschild and I still get the error, it must be `test_baumgarte_shapiro` which is failing because it is the only of the pair that uses Schwarzschild coordinates.  I feel like based on the TOV solution there's no way somewhere inside the star should be able to have M/R > .5, but apparently here it does.  I also locally checked what the `radius_squared` value is at the point with M/R>.5 and it's apparently  0.745597 which is greater than the final value of sqrt(.763), so I feel like something is definitely wrong.